guides/v2.1/coding-standards/docblock-standard-javascript.md

```diff
@@ -457,7 +459,7 @@ new&nbsp;project.MyClass()</pre></td>
   </tbody>
 </table>
 
-<h4 id="frontend-dev-guide-javascript-js-coding-docblock-typelang">JavaScript type language</h4>
+#### JavaScript type language
```

Comment:
Please append `{#frontend-dev-guide-javascript-js-coding-docblock-typelang}` to preserve the custom id.